src/main/resources/archetype-resources/pom.xml

```diff
@@ -73,31 +74,35 @@
         <version>${symbol_dollar}{tycho.version}</version>
         <configuration>
           <resolver>p2</resolver>
-          <pomDependencies>consider</pomDependencies>
-          <ignoreTychoRepositories>true</ignoreTychoRepositories>
+          <target>
+            <artifact>
+              <groupId>org.eclipse.jdt.ls</groupId>
+              <artifactId>org.eclipse.jdt.ls.tp</artifactId>
+              <version>${symbol_dollar}{jdt.ls.version}</version>
+            </artifact>
```

Comment:
Maybe we should create a separate target file (and module) for this. vscode-pde would then support building it, which currently, is not the case.

Reply:
I would very much be in favor of this if vscode-pde is able to handle it then.